packages/@ember/-internals/glimmer/lib/helpers/-resolve.ts

```diff
@@ -0,0 +1,38 @@
+/**
+@module ember
+*/
+import { Owner } from '@ember/-internals/owner';
+import { assert, runInDebug } from '@ember/debug';
+import { CapturedArguments } from '@glimmer/interfaces';
+import { createConstRef, isConstRef, valueForRef } from '@glimmer/reference';
+import { internalHelper } from './internal-helper';
+
+export default internalHelper(({ positional }: CapturedArguments, owner: Owner | undefined) => {
+  // why is this allowed to be undefined in the first place?
```

Comment:
@pzuraq This seemed strange. I suspect this is leftover from when you thought you didn't have to propagate owner in strict mode and left the `| undefined` in the signature, even though in practice it shouldn't ever be `undefined`?